src/style.css

```diff
@@ -0,0 +1,95 @@
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.color__snake {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex: 1 0 33%;
+
+  height: 300px;
+  width: 300px;
+  min-width: 300px;
+  min-height: 300px;
+}
+
+.block--num {
+  font-family: inherit;
+  font-size: 100px;
+  color: white;
+}
+
+.block:nth-child(1) {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.block:nth-child(2) {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.block:nth-child(3) {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.block:nth-child(4) {
```

Comment:
with correct classes `block--1, block--2, ...` you can remove `nth-child` from css